src/webview/view/components/forms/spfxProject/AdditionalStep.tsx

```diff
@@ -0,0 +1,60 @@
+import { VSCodeCheckbox } from '@vscode/webview-ui-toolkit/react';
+import * as React from 'react';
+import { StepHeader } from './StepHeader';
+import { PackageSelector } from './PackageSelector';
+
+interface AdditionalStepProps {
```

Comment:
Please leave two empty lines between imports and the rest of the code. 
I should probably add an eslint rule for that 🤔, Sorry my bad 😉

```suggestion
import { PackageSelector } from './PackageSelector';


interface AdditionalStepProps {
```

Reply:
Please let me know if you would like me to add the rule.